### PR TITLE
Update example

### PR DESCRIPTION
**Pull request by yasinghcse**
**Changes:**
- Update nodejs client examples
Screen Shot 2020-09-16 at 8 36 29 PM

- Update invoke action commands to use common flags for all examples

### CODE REVIEW

File: _layouts/documentation.html [hunk 1]
```diff
@@ -872,18 +872,14 @@ <h5>Creating and Invoking NodeJS actions</h5>
               </div>
             </li>
             <li style="list-style-type: decimal">
-              Invoke the <i>helloJS</i> action as a blocking activation:
+              Invoke an action <i>helloJS</i>:
               <div class="terminal">
-{% highlight bash %}$ wsk action invoke helloJS --blocking{% endhighlight %}
+{% highlight bash %}$ wsk action invoke helloJS -r -p name World{% endhighlight %}
```
Comment (rabbah) [inline _layouts/documentation.html:877]:
```suggestion
{% highlight bash %}$ wsk action invoke helloJS --result --param name World{% endhighlight %}
```

File: _layouts/documentation.html [hunk 2]
```diff
@@ -1087,7 +1083,7 @@ <h5>Creating And Invoking Python actions</h5>
             </li>
             <li style="list-style-type: decimal">Invoke the <i>helloPy</i> action using command-line parameters:
               <div class="terminal">
-{% highlight bash %}$ wsk action invoke helloPy --blocking --param name World{% endhighlight %}
+{% highlight bash %}$ wsk action invoke helloPy -r -p name World{% endhighlight %}
```
Comment (rabbah) [inline _layouts/documentation.html:1086]:
```suggestion
{% highlight bash %}$ wsk action invoke helloPy --result --param name World{% endhighlight %}
```

File: _layouts/documentation.html [hunk 3]
```diff
@@ -1212,7 +1208,7 @@ <h5>Creating and Invoking Java actions</h5>
             <li style="list-style-type: decimal">
               Invoke an action <i>helloJava</i>:
               <div class="terminal">
-{% highlight bash %}$ wsk action invoke helloJava --blocking --result{% endhighlight %}
+{% highlight bash %}$ wsk action invoke helloJava -r{% endhighlight %}
```
Comment (rabbah) [inline _layouts/documentation.html:1211]:
```suggestion
{% highlight bash %}$ wsk action invoke helloJava --result{% endhighlight %}
```

File: _layouts/documentation.html [hunk 4]
```diff
@@ -1290,7 +1286,7 @@ <h5>Creating and Invoking PHP actions</h5>
             <li style="list-style-type: decimal">
               Invoke an action <i>helloPHP</i>:
               <div class="terminal">
-{% highlight bash %}$ wsk action invoke helloPHP --blocking --result --param name World{% endhighlight %}
+{% highlight bash %}$ wsk action invoke helloPHP -r -p name World{% endhighlight %}
```
Comment (rabbah) [inline _layouts/documentation.html:1289]:
```suggestion
{% highlight bash %}$ wsk action invoke helloPHP --result --param name World{% endhighlight %}
```

File: _layouts/documentation.html [hunk 5]
```diff
@@ -1396,7 +1392,7 @@ <h5>Creating and Invoking Ruby actions</h5>
             <li style="list-style-type: decimal">
               Invoke an action <i>hello_ruby</i>:
               <div class="terminal">
-{% highlight bash %}$ wsk action invoke hello_ruby --blocking --result --param name World{% endhighlight %}
+{% highlight bash %}$ wsk action invoke hello_ruby -r -p name World{% endhighlight %}
```
Comment (rabbah) [inline _layouts/documentation.html:1395]:
```suggestion
{% highlight bash %}$ wsk action invoke hello_ruby --result --param name World{% endhighlight %}
```

**Review by rabbah — APPROVED** [review]
LGTM - Thank you.

I suggest using the longer --result and --param for clarity.

File: _layouts/documentation.html (hunk 1 above)
Comment (yasinghcse) [reply _layouts/documentation.html]: Done.

File: _layouts/documentation.html (hunk 2 above)
Comment (yasinghcse) [reply _layouts/documentation.html]: Added the changes under a new commit

File: _layouts/documentation.html (hunk 3 above)
Comment (yasinghcse) [reply _layouts/documentation.html]: Added the changes under a new commit

File: _layouts/documentation.html (hunk 4 above)
Comment (yasinghcse) [reply _layouts/documentation.html]: Added the changes under a new commit

File: _layouts/documentation.html (hunk 5 above)
Comment (yasinghcse) [reply _layouts/documentation.html]: Added the changes under a new commit

**Comment by yasinghcse** [comment]
All example updates to use --param and --result flag.

**Comment by rabbah** [comment]
Thank you @yasinghcse.

**Pull request merged**